Sources/Load.php

```diff
@@ -48,9 +48,13 @@ function reloadSettings()
 		$modSettings = array();
 		if (!$request)
 			display_db_error();
-		while ($row = $smcFunc['db_fetch_row']($request))
-			$modSettings[$row[0]] = $row[1];
+		$rows = $smcFunc['db_fetch_all']($request);
+		foreach ($rows as $row)
+		{
+			$modSettings[$row['variable']] = $row['value'];
+		}
 		$smcFunc['db_free_result']($request);
+		unset($rows);
 
```

Comment:
Do we need to assign to the temp $rows variable or can we just call a straight through?

Reply:
I don't understand your question.
What you mean with call straight through ( I understand the words but not what kind of coding style you want)

Reply:
```
		foreach ($smcFunc['db_fetch_all']($request) as $row)
```